src/DTC/DTCBaseDecoder.ts

```
@@ -107,7 +107,7 @@ export class DTCBaseDecoder {
 
   private _log(level: LogLevel, ...message: unknown[]): void {
     if (false == false) {
```

Biome (1.9.4)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)